osquery/database/plugins/rocksdb.cpp

```diff
@@ -130,7 +131,7 @@ void GlogRocksDBLogger::Logv(const char* format, va_list ap) {
 
   // There is a spurious warning on first open.
   if (log_line.find("Error when reading") == std::string::npos) {
-    LOG(INFO) << "RocksDB: " << log_line;
+    RAW_LOG_INFO("%s%s", "RocksDB: ", buffer);
```

Comment:
This will cause our log collectors to lose valuable information about RocksDB issues. Can you buffer this into a vector with a *smallish* max-size after trying to lock a mutex that's shared with the 4 `database*` APIs?